tests/dialogs/src/index.ts

```diff
@@ -0,0 +1,303 @@
+import { cardAttachment, MessageActivity } from '@microsoft/teams.api';
+import { App, HttpPlugin } from '@microsoft/teams.apps';
+import {
+  Card,
+  ICard,
+  SubmitAction,
+  TaskFetchAction,
+  TaskFetchData,
+  TextInput,
+} from '@microsoft/teams.cards';
+import { DevtoolsPlugin } from '@microsoft/teams.dev';
+import dotenv from 'dotenv';
+import path from 'path';
+
+dotenv.config({
+  path: path.join(__dirname, '..', 'env', '.env.local'),
+});
+
+const httpPlugin = new HttpPlugin();
+
+if (!process.env['BOT_ENDPOINT']) {
+  console.warn('No remote endpoing detected. Using webpages for dialog will not work as expected');
+}
+
+httpPlugin.get('/', (_req, res) => {
+  res.sendFile(path.join(__dirname, 'views', 'customform.html'));
+});
+
+const app = new App({
+  plugins: [new DevtoolsPlugin(), httpPlugin],
+});
+
+// :snippet-start: dialog-entry-point
+app.on('message', async ({ send }) => {
+  await send({ type: 'typing' });
+
+  // Create the launcher adaptive card
+  const card: ICard = new Card()
+    .withBody({
+      type: 'TextBlock',
+      text: 'Select the examples you want to see!',
+      size: 'large',
+      weight: 'bolder',
+    })
+    .addActions(
+      // raw action
+      {
+        type: 'Action.Submit',
+        title: 'Simple form test',
+        data: {
+          msteams: {
+            type: 'task/fetch',
+          },
+          opendialogtype: 'simple_form',
+        },
+      },
+      // Special type of action to open a dialog
+      new TaskFetchAction({})
+        .withTitle('Webpage Dialog')
+        // This data will be passed back in an event so we can
+        // handle what to show in the dialog
+        .withValue(new TaskFetchData({ opendialogtype: 'webpage_dialog' })),
+      new TaskFetchAction({})
+        .withTitle('Multi-step Form')
+        .withValue(new TaskFetchData({ opendialogtype: 'multi_step_form' })),
+      new TaskFetchAction({})
+        .withTitle('Mixed Example')
+        .withValue(new TaskFetchData({ opendialogtype: 'mixed_example' }))
+    );
+
+  // Send the card as an attachment
+  await send(new MessageActivity('Enter this form').addCard('adaptive', card));
+});
+// :snippet-end:
+
+/**
+// :snippet-start: dialog-open
+app.on('dialog.open', async ({ activity }) => {
```

Comment:
is this needed or an accidental leftover?

Reply:
It was originally intentionally there, but I'm moving it to docs directly now